examples/output_voltage_measurement/_constants.py

```diff
@@ -0,0 +1,6 @@
+"""Constants."""
+USE_SIMULATION = True
```

Comment:
It's weird that ni-python-styleguide didn't make you leave a blank line here.

Reply:
Included a blank line now.